docker/docker-compose.yml

```diff
@@ -22,7 +22,9 @@ services:
     logging: *default-logging
 
   lemmy:
-    image: dessalines/lemmy:1.0.0-alpha.4
```

Comment:
Not sure how this got added, but it meant no changes would ever get tested correctly in docker.